internal/pkg/image/driver/imagedriver.go

```diff
@@ -111,10 +120,43 @@ func (d *fuseappsDriver) Mount(params *image.MountParams, mfunc image.MountFunc)
 			// this will be passed as the first ExtraFile below, always fd 3
 			srcPath = "/proc/self/fd/3"
 		}
-		f.cmd = exec.Command(f.cmdPath, "-f", "-o", optsStr, srcPath, params.Target)
+		cmd = exec.Command(f.cmdPath, "-f", "-o", optsStr, srcPath, params.Target)
 
 	case "ext3":
-		return fmt.Errorf("mounting an EXT3 filesystem requires root or a suid installation")
+		f = &d.ext3Feature
+		srcPath := params.Source
+		if path.Dir(params.Source) == "/proc/self/fd" {
+			// this will be passed as the first ExtraFile below, always fd 3
+			srcPath = "/proc/self/fd/3"
+		}
+		optsStr := ""
+		if os.Getuid() != 0 {
+			// Bypass permission checks so all can be read,
+			//  especially overlay work dir
+			optsStr = "fakeroot"
+		}
+		if (params.Flags & syscall.MS_RDONLY) != 0 {
+			if optsStr != "" {
+				optsStr += ","
+			}
+			optsStr += "ro"
+		}
+		// Run fuse2fs through stdbuf to be able to read the
+		//  warnings sometimes sent through stdout
+		cmd = exec.Command("/usr/bin/stdbuf", "-oL", f.cmdPath, "-f", "-o", optsStr, srcPath, params.Target)
```

Comment:
Maybe use `bin.FindBin("stdbuf")` for consistency and avoid hardcoded path

Reply:
Ok done